apps/els_lsp/src/els_general_provider.erl

```diff
@@ -154,7 +154,17 @@ server_capabilities() ->
                 renameProvider =>
                     els_rename_provider:is_enabled(),
                 callHierarchyProvider =>
-                    els_call_hierarchy_provider:is_enabled()
+                    els_call_hierarchy_provider:is_enabled(),
+                semanticTokensProvider =>
```

Comment:
In the case Wrangler is disabled, is this going to be equivalent to a `semanticTokensProvider => false` or do we get extra traffic?

Reply:
Yes, wls_handler returns false if Wrangler is not enabled in the config, there are no outgoing calls.

Reply:
What I meant by "traffic" is communication between the client (editor) and the LS. In other words, is the following (which we would be sending in case Wrangler is disabled):

```
semanticTokensProvider => #{
  legend =>
    #{
       tokenTypes => [],
       tokenModifiers => []
    },
    range => false,
   full => true
}
```                    

Equivalent to:

```
semanticTokensProvider => false
```
 
? Or would client start sending requests to the server that would result in empty responses?

Reply:
Right, I fixed this, the is_enabled() function had not been used. Now the client does not request semantic tokens, I also tested it.

Reply:
According to the protocol, the semanticTokensProvider property can not be a boolean value: https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#semanticTokensOptions